src/drawing/DrawingTarget.scss

```diff
@@ -3,5 +3,9 @@
 
     &:hover {
```

Comment:
Should this apply to focus too?

Reply:
When focus, we explicitly add `is-active` class, because IE doesn't focus